lib/src/main/java/org/team100/lib/motion/drivetrain/kinodynamics/SwerveKinodynamicsFactory.java

```diff
@@ -35,7 +35,7 @@ public static SwerveKinodynamics get() {
                 // TODO: make tests specify kinodynamics instead.
                 return new SwerveKinodynamics(4, 2, 3, 13, 20 * Math.PI, 0.5, 0.5, 0.3);
             case BETA_BOT:
-                return new SwerveKinodynamics(4, 2, 3, 13, 20 * Math.PI, 0.491, 0.765, 0.3);
+                return new SwerveKinodynamics(5, 5, 4, 13, 20 * Math.PI, 0.491, 0.765, 0.3);
```

Comment:
the third number in this list is max deceleration which should certainly be higher than max acceleration.